docs/diff/rules.md

```diff
@@ -0,0 +1,190 @@
+# Rules for Diff Tool
+
+## Rules to categorize differences
+
+In rules, a difference can be categorized as Breaking or Non-Breaking.
```

Comment:
Suggest changing sentence to:
The Diff Tool rule categorizes differences as Breaking or Non-Breaking.